examples/web/index.html

```diff
@@ -18,11 +18,28 @@
       );
       const go = new Go();
-      WebAssembly.instantiateStreaming(fetch("runme.wasm"), go.importObject).then((runme) => {
-        go.run(runme.instance);
-        console.log(GetSnippets(markdown));
-        console.log(GetDocument(markdown));
-        console.log(PrepareScript(["bash -c 'echo \"1\"'"]));
+      WebAssembly.instantiateStreaming(fetch("runme.wasm"), go.importObject).then((obj) => {
+        go.run(obj.instance);
+
+        console.log("return error", Runme.getSource());
```

Comment:
Technically it's odd to "return" errors in JS, but let's not get hung up on it. Lets plan to iterate on errors as part of building out the editing stuff.

Reply:
There is no other way AFAIK. I could return a string but I guess it makes sense - an exported JS function returns a JS `Error` instance. Or I could return `null` :) We will get back to it.